pyminio/pyminio_test_class.py

```diff
@@ -1,42 +1,30 @@
 import os
-from os.path import normpath, join
-from main import Pyminio
+from .main import Pyminio
 
+TEST_DIR = "/test/"
+
+
+class PyminioTest(Pyminio):
```

Comment:
make a directory of tests and put it inside a `utils` module

Reply:
d